Cargo.toml

```diff
@@ -138,7 +138,7 @@ email_address = "0.2.4"
 enum-as-inner = "0.6"
 enum-map = "2.6.3"
 env_logger = "0.10"
-flate2 = "1.0.24"
+flate2 = "1.0.24" # TODO features = ["zlib-ng"], default-features = false ?
```

Comment:
Should we consider the C zlib library instead of the (slower) rust one? This is from the flate2 docs

Reply:
Note that we tested the speed of the rust implementation and it's very similar to the current brotli implementation.